client/components/map/StickyFooter.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import { Box, Flex } from "@chakra-ui/react";
+
+const StickyFooter = () => {
+	const [isExpanded, setIsExpanded] = useState(false);
+
+	const handleExpand = () => {
+		setIsExpanded(!isExpanded);
+	};
+
+	return (
+		<>
+			<Box
+				as="footer"
+				w="100%"
+				bg="white"
+				py={4}
+				position="fixed"
+				bottom={isExpanded ? "92vh" : 0}
+				transition="bottom 0.3s"
+				zIndex={9999}
+			>
+				<Box textAlign="center" position="relative" cursor="pointer" onClick={handleExpand}>
+					<Box
+						position="absolute"
+						top="-10px"
+						left="50%"
+						transform="translateX(-50%)"
+						w="30px"
+						h="2px"
+						bg="black"
+					/>
+					<Box>
+						<p>Your Footer Text</p>

```

Comment:
is there actual footer text that supposed to go here?

Reply:
Removing this component as it is unused too